opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -130,8 +130,10 @@ message Metric {
     // and `sum` fields.
     // 3. New receivers, which are aware of this change MUST convert these into
     // `gauge` and `sum` by using the provided as_int field in the oneof values.
+    // This field will be removed in ~3 months, on July 1, 2021.
```

Comment:
Is this enough time for SDKs THEN collector?

Reply:
Do you think 6 months will be better?

Reply:
I imagine the whole project is better off if we move quickly. I support 3 months.

Reply:
We discussed this in the SiG.  I'll open an OTEP around attempting to use real-usage to determine migration timelines vs. hard dates and we can discuss my concerns on that.   This is a non-blocking concern on this PR.